packages/ui/src/components/RadioGroup/style.less

```diff
@@ -0,0 +1,42 @@
+@import (reference) "@open-condo/ui/src/tokens/variables.less";
+@import (reference) "@open-condo/ui/src/components/style/mixins/typography";
+
+.condo-radio-group {
+  display: flex;
+  flex-direction: column;
+
+  & > .condo-radio-group-container {
+    display: flex;
+    flex-direction: column;
+
+    & > .condo-space {
+      display: none;
+      margin-top: @condo-global-spacing-16;
+      padding-left: @condo-global-spacing-16;
+    }
+
+    & > .condo-radio-group-title {
+      display: flex;
+      align-items: center;
+      cursor: pointer;
+
+      & span:first-child {
```

Comment:
it would not work, if I'll pass non-condo icon (pure svg or smith like that).
wrapping icon in classnamed span is decent option for that